pipeline/stage02_processing/configs/config_LENS.yaml

```diff
@@ -46,10 +54,10 @@ NORMALIZE_BY: 'max'
 # function by elephant
 HIGHPASS_FREQ: 0  # in Hz
 LOWPASS_FREQ: 5  # in Hz
-FILTER_ORDER: 2
+FILTER_ORDER: 6 #2
 # filter function used in scipy backend.
 # options: ‘filtfilt’, 'lfilter’, ‘sosfiltfilt’
-FILTER_FUNCTION: 'filtfilt'
+FILTER_FUNCTION: 'lfilter' #'filtfilt'
```

Comment:
why?

Reply:
Sorry, I was just curious about how the filter function affects the output and forgot to change it!